lib/plugins/EPrints/Plugin/Import/PubRouter.pm

```diff
@@ -253,14 +249,7 @@ sub xml_to_epdata
 		elsif( $sourceid_type eq "isbn" )
 		{
 			$epdata->{isbn} = $plugin->xml_to_text( $sourceid );
-		}
-		
-		#Add all types of source_id values to Additional Information field
-		push @sourceid_notes, "$sourceid_type " . $plugin->xml_to_text( $sourceid );
-	}
-	if( scalar @sourceid_notes > 0 )
-	{
-		$epdata->{note} = $plugin->appendToNote( $epdata->{note}, "Journal IDs", join( "; ", @sourceid_notes ) );
+		}	
```

Comment:
I just noticed line 245: is there a missing `!` before $issn_set?  Otherwise the pissn would overwrite the eissn value.

If that is the case, then should you combine lines 241 - 248:
```
elsif( ! $issn_set && ($sourceid_type eq "issn" || $sourceid_type eq "pissn"))
{
	$epdata->{issn} = $plugin->xml_to_text( $sourceid );
}
```

Reply:
See fix in https://github.com/jisc-services/eprints-plugin-PubRouter-RIOXXplus-Connector/pull/5/commits/5d2140888ed57771e75f7fe23225da064962ed62